app/controllers/plug/site_notice_controller.rb

```diff
@@ -0,0 +1,22 @@
+require_dependency 'plug/application_controller'
+
+module Plug
+  class SiteNoticeController < ApplicationController
+    if Rails.version.to_i < 5
```

Comment:
Ugrh no that is annoying.

I'm not sure if we can do much about this. until we drop support for older rails versions.

Reply:
The callback doesn't call any actions.